particles/Tutorial/Kotlin/5_Collections/Collections.kt

```diff
@@ -0,0 +1,42 @@
+package arcs.tutorials
+
+import arcs.*
+import kotlin.native.internal.ExportForCppRuntime
+
+/**
+ * Sample Kotlin-WASM Particle to use a JSON store.
+ */
+class CollectionsParticle : Particle() {
+
+   private val people = Collection { CollectionsParticle_InputData() }
+   init {
```

Comment:
nit: I would place a blank line between the instance variables and the init block